src/main/java/com/it4045/pokemonviewer/enterprise/service/PokemonServiceStub.java

```diff
@@ -23,13 +23,19 @@
      */
     @Override
     public Pokemon fetchByName(String pokemonName) {
-        pokemon.setPokemonName("Charizard");
-        pokemon.setPokemonId(1);
+        pokemon.setName("Charizard");
+        pokemon.setId(1);
+        pokemon.setAvgSpawns(0.31);
         return pokemon;
     }
 
     @Override
     public List<Pokemon> fetchPokemons() throws IOException {
         return pokemonDAO.fetchAllPokemons();
     }
+
+    @Override
+    public void delete(int id) {
+
+    }
```

Comment:
This looks like the start of a new feature.  Avoid adding new features in a code review, as a good code review should merge to master, and only features that meet the team's Definition of Done should be merged to master.  If functionality is introduced and not completed, it adds to technical debt, as it's unused code.